packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -385,6 +385,7 @@ export class DefaultPageBlockComponent
           </div>
           ${childrenContainer}
         </div>
+        <affine-drag-indicator></affine-drag-indicator>
```

Comment:
IMO it is an anti-pattern to putting `affine-drag-indicator` outside of `drag-handle`

Reply:
The cost of each components with dragging behavior maintaining their own `affine-drag-indicator`, can you suggest a better way to resolve this?